devices/tplink_re200-v4.yml

```diff
@@ -0,0 +1,23 @@
+vendor: TP-Link
+model: RE200
+variant: v4
+bootloader: unknown
+cpu: MediaTek MT7628AN
+cpu_cores: 1
+cpu_mhz: 580
+memory_mb: 64
+storage:
+  - type: unknown
+    size_mb: 8
+device_type: WiFi Router
+ethernet_100m_ports: 1
+jtag: false
+serial: 57600 / 8N1
```

Comment:
Maybe one can split this into two variables, but I'm not really sure whether that's an improvement yet.

Reply:
We can apply the same as for storage here.

Reply:
I did not find any meaningful variation in 8N1 settings so I just kept it as a enum of strings.